src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
 
-export const TodoModal: React.FC = () => {
+type Props = {
+  selectedTodo: Todo | undefined,
+  setInicializationModal: (inicializationModal: boolean) => void,
+  setTodoModal: (selectedTodo: Todo | undefined) => void,
+  setUser: (user: User | null) => void,
+  user: User | null
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  setInicializationModal,
+  setTodoModal,
+  user,
+  setUser,
+}) => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {user === null ? (
```

Comment:
Use logical operators 
```suggestion
      {!user ? (
```